src/connections.jl

```diff
@@ -507,7 +528,11 @@ function conninfo(jl_conn::Connection)
     ci_ptr = libpq_c.PQconninfo(jl_conn.conn)
```

Comment:
Does this function need locking?

Reply:
I don't believe so, as the pointer returned is a copy initialized from a static default buffer. However, I should probably check for a closed connection first.

Reply:
the buffer is never editted after the `Connection` object is fully constructed?

Reply:
Confirmed never editted,